Edited Makefile install routine as it had the CURDIR and
DESTDIR variables transposed.
--- a/Makefile
+++ b/Makefile
@@ -45,7 +45,7 @@ strip:
 
 install: all
 	install -d $(DESTDIR)$(BINDIR)
-	install -m 755 $(PROG) $(BINDIR)$(DESTDIR)
+	install -m 755 $(PROG) $(DESTDIR)$(BINDIR)
 
 love:
 	@echo "Not right now, I have a headache."
